default/python/AI/ProductionAI.py

```diff
@@ -315,7 +315,7 @@ def generate_production_orders():
                         cost, time = empire.productionCostAndTime(production_queue[production_queue.size - 1])
                         building_expense += cost / time
 
-            if (total_pp > 40 or ((current_turn > 40) and (ColonisationAI.empire_status.get('industrialists', 0) >= 20))) and ("BLD_INDUSTRY_CENTER" in possible_building_types) and ("BLD_INDUSTRY_CENTER" not in (capital_buildings+queued_building_names)) and (building_expense < building_ratio*total_pp):
+            if (total_pp > 40 or ((current_turn > 40) and (state.population_with_industry_focus() >= 20))) and ("BLD_INDUSTRY_CENTER" in possible_building_types) and ("BLD_INDUSTRY_CENTER" not in (capital_buildings+queued_building_names)) and (building_expense < building_ratio*total_pp):
```

Comment:
Seeing these lines makes me think I'd like to add a TODO here, to broaden this somewhat so that it could take into account any captured Industrial Centers (i.e., for completed buildings look not only at the Capital).  It's not really related to the subject of this PR, but if I just go directly add that TODO now it would create a conflict with the merger of this, so I'm noting this comment and you can decide if you want to go ahead and add a commit here for the TODO or else we can just try to remember this later.

Reply:
I pushed a separate commit with the TODO comment